### PR TITLE
[pt] Unsupervised learning

### PR DESCRIPTION
**Pull request by gabriel19913**
I just translated the Unsupervised learning to portuguese. @leportella can you please review it?

### CODE REVIEW

**Comment by shervinea** [comment]
Thank you for your work on this @gabriel19913. Really appreciated!

**Comment by shervinea** [comment]
@gabriel19913 Thank you again for your work! It seems that everyone is busy at the moment; please feel free to ping other people around, if you know any other native speaker.

**Comment by gabriel19913** [comment]
Ok @shervinea I will look for someone to help.

File: pt/cheatsheet-unsupervised-learning.md [hunk 1]
```diff
@@ -1,340 +1,339 @@
...
 
-&#10230;
+&#10230; Motivação ― O objetivo do aprendizado não supervisionado é encontrar padrões em dados sem rótulo {x(1),...,x(m)}.
 
 <br>
 
 **4. Jensen's inequality ― Let f be a convex function and X a random variable. We have the following inequality:**
 
-&#10230;
+&#10230; Desigualdade  de Jensen's ― Seja f um função convexa e X uma variável aleatória. Temos a seguinte desigualdade:
```
Comment (TiagoDanin) [inline pt/cheatsheet-unsupervised-learning.md]: - Seja f **uma** função

File: pt/cheatsheet-unsupervised-learning.md [hunk 2]
```diff
@@ -1,340 +1,339 @@
...
 
-&#10230;
+&#10230; diagonal
 
 <br>
 
 **35. Remark: the eigenvector associated with the largest eigenvalue is called principal eigenvector of matrix A.**
 
-&#10230;
+&#10230; Observação: o autovetor associado com o maior autovalor é chamado de autorvetor principal da matriz A.
```
Comment (TiagoDanin) [inline pt/cheatsheet-unsupervised-learning.md]: - Observação: o autovetor associado com o maior autovalor é chamado de **autovetores** principal da matriz A.

File: pt/cheatsheet-unsupervised-learning.md [hunk 3]
```diff
@@ -1,340 +1,339 @@
...
 
 <br>
 
-**36. Algorithm ― The Principal Component Analysis (PCA) procedure is a dimension reduction technique that projects the data on k
-dimensions by maximizing the variance of the data as follows:**
+**36. Algorithm ― The Principal Component Analysis (PCA) procedure is a dimension reduction technique that projects the data on k dimensions by maximizing the variance of the data as follows:**
 
-&#10230;
+&#10230; Algoritmo ― O processo de Análise de Componente Principal (PCA) é uma técnica de redução de dimensção que projeta os dados em dimensções k ao maximizar a variância dos dados como se segue:
```
Comment (TiagoDanin) [inline pt/cheatsheet-unsupervised-learning.md]: - O processo de Análise de Componente Principal (PCA) é uma técnica de redução da **dimensão** que projeta os dados em **dimensões** k ao maximizar a variância dos dados como se segue:

File: pt/cheatsheet-unsupervised-learning.md [hunk 4]
```diff
@@ -1,340 +1,339 @@
...
 
-&#10230;
+&#10230; Etapa 4: Projetar os dados em spanR(u1,...,uk).
 
 <br>
 
 **41. This procedure maximizes the variance among all k-dimensional spaces.**
 
-&#10230;
+&#10230; Esse processo maximiza a variância entre todos espaços dimensionais k.
```
Comment (TiagoDanin) [inline pt/cheatsheet-unsupervised-learning.md]: - Esse processo maximiza a variância entre todos **os** espaços dimensionais k.

File: pt/cheatsheet-unsupervised-learning.md [hunk 5]
```diff
@@ -1,340 +1,339 @@
...
 
-&#10230;
+&#10230; [Dados em espaço característico, Encontrar componentes principais, Dados no espaço de componentes principais]
 
 <br>
 
 **43. Independent component analysis**
 
-&#10230;
+&#10230; Análise de componente independete
```
Comment (TiagoDanin) [inline pt/cheatsheet-unsupervised-learning.md]: - Análise de componente **independente**

**Comment by gabriel19913** [comment]
@TiagoDanin I did the review as you suggest, I just have a doubt in one item. Do you approve the review?
@shervinea After the review approve I will update the contributors file.

File: pt/cheatsheet-unsupervised-learning.md (hunk 2 above)
Comment (TiagoDanin) [reply pt/cheatsheet-unsupervised-learning.md]: OPS! Correct is **autovetor**

**Comment by gabriel19913** [comment]
@shervinea I think we are good to merge this PR.
I already added the Thiago name into the Contributors file.
@TiagoDanin thanks for the review!

**Comment by shervinea** [comment]
Thank you both very much @gabriel19913 and @TiagoDanin! Merging this PR now.

**Pull request merged**